meta/main.yml

```diff
@@ -0,0 +1,13 @@
+galaxy_info:
+  author: Andrew Euredjian
+  description: Role to setup Kernel dump files
+  company: Red Hat, Inc.
+  license: GPLv3
+  min_ansible_version: 2.4
+  platforms:
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - oasis
```

Comment:
I'm not sure why you pulled out
``yaml
dependencies: []
``
from here, but it's considered best practices to leave that in and have it always be blank.